examples/jsm/loaders/VTKLoader.js

```diff
@@ -890,44 +890,7 @@ class VTKLoader extends Loader {
 
 			// Main part
 			// Get Dom
-			var dom = null;
-
-			if ( window.DOMParser ) {
-
-				try {
-
-					dom = ( new DOMParser() ).parseFromString( stringFile, 'text/xml' );
-
-				} catch ( e ) {
-
-					dom = null;
-
-				}
-
-			} else if ( window.ActiveXObject ) {
-
-				try {
-
-					dom = new ActiveXObject( 'Microsoft.XMLDOM' ); // eslint-disable-line no-undef
-					dom.async = false;
-
-					if ( ! dom.loadXML( /* xml */ ) ) {
-
-						throw new Error( dom.parseError.reason + dom.parseError.srcText );
-
-					}
-
-				} catch ( e ) {
-
-					dom = null;
-
-				}
-
-			} else {
-
-				throw new Error( 'Cannot parse xml string!' );
-
-			}
+			var dom = new DOMParser().parseFromString( stringFile, 'application/xml' );
```

Comment:
In other loaders like `SVGLoader` or `3MFLoader` we use `DOMParser` without any fallbacks or error testing.